backend/app/rest/telegram.go

```diff
@@ -0,0 +1,63 @@
+package rest
```

Comment:
to me this file doesn't seem to be a logical part of rest package as it has very little to do with rest

Reply:
Agreed, however, I have no better idea where to put it: it should not be part of notify package as it will be cut into a separate package. Suggestions are welcome.